### ISSUE TITLE
the new pytorch.bin  is bigger than original model issue

### ISSUE THREAD

**Issue by lb553024300** [horseee/LLM-Pruner#37]
When I choose save model, I found some strange things。The new pytorch.bin  is bigger than original model。I choose Baichuan-7B ,--pruning_ratio 0.5 for test, and add  --save_model  for save the model after pruning. but the new pytorch.bin is 17GB, and the  original model is only 13GB? 
Could you please tell me why? Thank you!

**Comment by VainF** [comment]
Hi @lb553024300, don't worry about the bin size. We store the whole model object into the disk with ``torch.save(model)``, so the saved file will be larger than ``torch.save(model.state_dict())``

**Comment by lb553024300** [comment]
> Hi @lb553024300, don't worry about the bin size. We store the whole model object into the disk with `torch.save(model)`, so the saved file will be larger than `torch.save(model.state_dict())`

Get it,thanks

**Comment by horseee** [comment]
Hi. Could you please check if you deleted the gradient for calculating Taylor before saving the model?

**Comment by trinhdoduyhungss** [comment]
Hi @VainF, can you tell me how to convert it back to the same format as the original model? The lighter capacity is what I need for storage. Thank you very much

> Hi @lb553024300, don't worry about the bin size. We store the whole model object into the disk with `torch.save(model)`, so the saved file will be larger than `torch.save(model.state_dict())`
